net/ddns-scripts/files/usr/share/ddns/default/infomaniak.com.json

```diff
@@ -0,0 +1,7 @@
+{
+	"name": "infomaniak.com",
+	"ipv4": {
+		"url": "https://[USERNAME]:[PASSWORD]@infomaniak.com/nic/update?hostname=[DOMAIN]&myip=[IP]",
+		"answer": "good|nochg"
+	}
+}
```

Comment:
Is there no ipv6 support?

Reply:
Yes they do. Well spotted, I just added it.